read-fonts/src/tables/name.rs

```diff
@@ -238,3 +238,54 @@ static MAC_ROMAN: [u16; 128] = [
     63743, 210, 218, 219, 217, 305, 710, 732, 175, 728, 729, 730, 184, 733,
     731, 711,
 ];
+
+/// A lookup pairing (sorted) unicode values to Mac Roman values
+#[rustfmt::skip]
+static MAC_ROMAN_ENCODE: [(u16, u8); 128] = [
+    (160, 202), (161, 193), (162, 162), (163, 163),
+    (165, 180), (167, 164), (168, 172), (169, 169),
+    (170, 187), (171, 199), (172, 194), (174, 168),
+    (175, 248), (176, 161), (177, 177), (180, 171),
+    (181, 181), (182, 166), (183, 225), (184, 252),
+    (186, 188), (187, 200), (191, 192), (192, 203),
+    (193, 231), (194, 229), (195, 204), (196, 128),
+    (197, 129), (198, 174), (199, 130), (200, 233),
+    (201, 131), (202, 230), (203, 232), (204, 237),
+    (205, 234), (206, 235), (207, 236), (209, 132),
+    (210, 241), (211, 238), (212, 239), (213, 205),
+    (214, 133), (216, 175), (217, 244), (218, 242),
+    (219, 243), (220, 134), (223, 167), (224, 136),
+    (225, 135), (226, 137), (227, 139), (228, 138),
+    (229, 140), (230, 190), (231, 141), (232, 143),
+    (233, 142), (234, 144), (235, 145), (236, 147),
+    (237, 146), (238, 148), (239, 149), (241, 150),
+    (242, 152), (243, 151), (244, 153), (245, 155),
+    (246, 154), (247, 214), (248, 191), (249, 157),
+    (250, 156), (251, 158), (252, 159), (255, 216),
+    (305, 245), (338, 206), (339, 207), (376, 217),
+    (402, 196), (710, 246), (711, 255), (728, 249),
+    (729, 250), (730, 251), (731, 254), (732, 247),
+    (733, 253), (937, 189), (960, 185), (8211, 208),
+    (8212, 209), (8216, 212), (8217, 213), (8218, 226),
+    (8220, 210), (8221, 211), (8222, 227), (8224, 160),
+    (8225, 224), (8226, 165), (8230, 201), (8240, 228),
+    (8249, 220), (8250, 221), (8260, 218), (8364, 219),
+    (8482, 170), (8706, 182), (8710, 198), (8719, 184),
+    (8721, 183), (8730, 195), (8734, 176), (8747, 186),
+    (8776, 197), (8800, 173), (8804, 178), (8805, 179),
+    (9674, 215), (63743, 240), (64257, 222), (64258, 223),
+];
+
+#[cfg(test)]
+mod tests {
+    use super::*;
+
+    #[test]
+    fn mac_roman() {
+        static INPUT: &str = "Joachim Müller-Lancé";
+        for c in INPUT.chars() {
+            let enc = MacRomanMapping.encode(c).unwrap();
+            assert_eq!(MacRomanMapping.decode(enc), c);
```

Comment:
To confirm, this test failed before and now passes?

Reply:
correct.